### ISSUE TITLE
stations.yml I see and can can select the stations but not start the selected radio

### ISSUE THREAD

**Issue by spider7611** [milaq/YCast#84]
Hi,

I setting up my stations.yml and see and can select the stations but the selected radio doesn't start.
If I browse in Radiobrowser folder and select the same station it is working good. 
Waht is my problem with my stations.yml ?

`Magyar:
  Radio1 Debrecen: http://91.82.85.44:8080/debrecenradiofm95
  Radio1 Budapest: https://stream.radio1.hu/high.mp3
  Retro Radio: https://stream.retroradio.hu/high.mp3

Electronic:
  TechnoBase.FM: https://listener1.mp3.tb-group.fm/tb.mp3
  Dance UK: https://dancestream.danceradiouk.com/;
  Costa Del Mar Deep House: https://radio4.cdm-radio.com:18012/stream-mp3-Deep
  Dance Wave1: http://78.31.65.20:8080/dance.mp3
  Minimal Mix Radio Stream: http://5.39.71.159:8750/media
  DnBRadio.com: http://46.105.124.104:8000/dnbradio_main.mp3
  Bassdrive: http://50.7.98.106:8200
  SomaFM Fluid: http://ice1.somafm.com/fluid-128-mp3

Chillout:
  Lounge-Radio.com: https://fr1.streamhosting.ch/lounge128.mp3
  Joint Radio: http://radio.jointil.net:9998
  SomaFM DEF CON Radio: http://ice1.somafm.com/defcon-256-mp3
  SomaFM Drone Zone: http://ice1.somafm.com/dronezone-256-mp3
  SomaFM Mission Control: http://ice1.somafm.com/missioncontrol-128-mp3
  The Jazz Groove: http://west-mp3-128.streamthejazzgroove.com
  Radionomy Downbeat: http://streaming.radionomy.com/TempoOfTheDownbeat1`

**Comment by TheBossME** [comment]
Please Check your Stations file with a yamlchecker 

Like https://yamlchecker.com

**Comment by spider7611** [comment]
> Please Check your Stations file with a yamlchecker
> 
> Like https://yamlchecker.com

Tryed it and get this: Valid YAML!

**Comment by TheBossME** [comment]
Please Try with Leading Blanks in subregion ...

Chillout:
  Lounge-Radio.com: https://fr1.streamhosting.ch/lounge128.mp3
  Joint Radio: http://radio.jointil.net:9998
  SomaFM DEF CON Radio: http://ice1.somafm.com/defcon-256-mp3

**Comment by TheBossME** [comment]
![Uploading 103867577...]

**Comment by spider7611** [comment]
> Please Try with Leading Blanks in subregion ...
> 
> Chillout:
> Lounge-Radio.com: https://fr1.streamhosting.ch/lounge128.mp3
> Joint Radio: http://radio.jointil.net:9998
> SomaFM DEF CON Radio: http://ice1.somafm.com/defcon-256-mp3

Sorry but don't know what you mean.

**Comment by spider7611** [comment]
![ycast](https://user-images.githubusercontent.com/45290475/103984689-e2a79380-5187-11eb-875c-1bd2316acee8.jpg)
This is what is look like. What I told I see my station and can browse and can select. Only the problem isnt't start on my receiver.

**Comment by TheBossME** [comment]
Is this Problem only with the stations file ?.
can you browse and Play in Country etc...?

**Comment by spider7611** [comment]
Yes. I told this in the first post. In the Radiobrowser folder everything working fine.

**Comment by TheBossME** [comment]
I have actually no idea.. should Made a Zoom Session to See more about your Setup.

**Comment by 915461** [comment]
Hi guys, did you get somewhere?
I have exactly the same situation, can assist in debugging if that helps.

**Comment by 915461** [comment]
One additional data point.
It doesn't work with **Denon Remote App** (D+M Group) 1.1.9 (2020-03-23) on **Android**
It **does** work with **Receiver AV** (Roy Gore) 1.6.62.0 running on **Windows 10**

**Comment by ecltony** [comment]
Hi guys,
Recently I start to have a similar problem. When I launch for the first time a station on my device throughout stations.yml file it is working. But when I want to change it again from on my device throughout stations.yml file, nothing happens The previous station keeps playing. If I want to change, I have to reboot my device (by unplugging it from the alimentation).

My YCast v 1.1.0 is installed on Synology DS214play with DSM 192.168.2.77. The DNS is managed by ASUS router RT-AC68U. The HiFI equipment tested is Denon Ceol N9.

Before, It worked, and I think since the last DSM update it only work like this...

**Comment by Essoloani** [comment]
> 
> 
> One additional data point.
> It doesn't work with **Denon Remote App** (D+M Group) 1.1.9 (2020-03-23) on **Android**
> It **does** work with **Receiver AV** (Roy Gore) 1.6.62.0
> running on **Windows 10**

A beta version of ReceiverAV exists for Android.

**Comment by spider7611** [comment]
> Hi guys,
> Recently I start to have a similar problem. When I launch for the first time a station on my device throughout stations.yml file it is working. But when I want to change it again from on my device throughout stations.yml file, nothing happens The previous station keeps playing. If I want to change, I have to reboot my device (by unplugging it from the alimentation).
> 
> My YCast v 1.1.0 is installed on Synology DS214play with DSM 192.168.2.77. The DNS is managed by ASUS router RT-AC68U. The HiFI equipment tested is Denon Ceol N9.
> 
> Before, It worked, and I think since the last DSM update it only work like this...

Hi, is it working for you now? I have nearly the same setup and want it to use it today and not working at all. Don't know what is changed.

**Comment by Essoloani** [comment]
I have this problem when I add a radio to my AVR's favorites. The idea is to be able to access the radio in case of unavailability of pi. Under these conditions I must do a reset of the AVR

**Comment by ecltony** [comment]
Hi, I didn't pay attention if it is related to the AVR's favorite addition, but I realized that the port 80 was not free anymore. I guess the update of Synology DS somehow modified mustache files...

So I reinstalled everything from scratch... And now it is working fine!

**Comment by spider7611** [comment]
What you reinstall? The whole DSM? I Don't want to do that.

**Comment by ecltony** [comment]
No, not all the DSM...
I reinstalled Python, all the packages etc... See my post here, for the detailed instructions :  https://www.avforums.com/threads/vtuner-internet-radio-stop-working-marantz-denon-3-case.2269858/page-7#post-29147130

Hope it will help
